### PR TITLE
Line breaks in comments + bug fix

### LINKED ISSUE #207: [New addon] Recover '\n' in comments
**Describe the new addon**

So you know that line breaks are impossible in comments? No, not really. They get somehow represented as a space, but acutally they still have the character code 10 instead of 32 like a regular space. I really don't know why they're not displayed since they're still there but I think that we should be able to somehow recover them.

I found that out by accident when I looked at some stuff I can do with comments. Every comment ends with a '\n' and that really confused me. So I made some more test and it seems like I'm right.

**Link to the new addon**

Doesn't exist.

~~Oh, btw, could that be enabled by default as a fix?~~

### PR DESCRIPTION
**Pull request by TheColaber**
**Resolves**

Resolves #207 

**Changes**

Added the addon.
Another useless-useful addon if you know what I mean
I'm back creating PRs for issues made ages ago :P
Oops! Accidentally pushed my quicc fix to this branch... Oh well... it fixed this anyway:
image

**Reason for changes**

It's just neat to see line breaks in comments.

**Tests**

It works I guess.

Note: **MOST OF THIS TEXT WAS STOLEN FROM #209! THAT ISSUE DOES NOT HAVE ANY COPYRIGHT SO YOU MAY NOT ACCUSE ME OF STEELING. END OF ARGUMENT.**

### CODE REVIEW

**Comment by RedGuy12** [comment]
well you can at least give the people who helped with #209 credit... copyright is automatic, there doesn't need to be a notice on the material or a form to fill out...

**Comment by TheColaber** [comment]
> well you can at least give the people who helped with #209 credit... copyright is automatic, there doesn't need to be a notice on the material or a form to fill out...

lol you probably read it wrong :P... The text that is written in my original post was taken from that PR.

**Comment by RedGuy12** [comment]
> > well you can at least give the people who helped with #209 credit... copyright is automatic, there doesn't need to be a notice on the material or a form to fill out...
> 
> lol you probably read it wrong :P... The text that is written in my original post was taken from that PR. it was a joke if you didn't notice

Oh haha you were talking about the post text I thought you were talking about the code lol

File: addons/comments-linebreaks/addon.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "credits": [
+    {
+      "name": "TheColaber",
+      "link": "https://scratch.mit.edu/users/TheColaber"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/users/*", "https://scratch.mit.edu/projects/*"],
```
Comment (WorldLanguages) [inline addons/comments-linebreaks/addon.json:13]: Why not studios?

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**